controllers/check-details.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var util = require('util');
+var _ = require('underscore');
+
+var Controller = require('hmpo-form-wizard').Controller;
+var Model = require('../models/email');
+
+var Submit = function Submit() {
+  Controller.apply(this, arguments);
+};
+
+util.inherits(Submit, Controller);
+
+Submit.prototype.saveValues = function saveValues(req, res, callback) {
+  var data = _.pick(req.sessionModel.toJSON(), _.identity);
+  var model = new Model(data);
+
+  // set template
+  switch (req.originalUrl) {
+    case '/permit-delivery/check-details':
+      model.set('template', 'permit');
+      break;
+    case '/permit-error/check-details':
+      model.set('template', 'error');
+      break;
+    case '/permit-lost-or-stolen/check-details':
```

Comment:
I'm not sure I like the way these urls are hard-coded in the controller. At the least I think the switch should partially match cases, but even better would be a centralised point that maintained the routes.